src/main/webapp/js/user-page-loader.js

```diff
@@ -38,14 +38,31 @@ function showMessageForm() {
         return response.json();
       })
       .then((loginStatus) => {
-        if (loginStatus.isLoggedIn) {
-          const messageForm = document.getElementById('message-form');
-          messageForm.action = '/messages?recipient=' + parameterUsername;
-          messageForm.classList.remove('hidden');
+        if (loginStatus.isLoggedIn && loginStatus.username == parameterUsername) {
+          fetchImageUploadUrlAndShowForm()
+          document.getElementById('about-me-form').classList.remove('hidden');
+        } else if (loginStatus.isLoggedIn) {
+          fetchImageUploadUrlAndShowForm()
         }
       });
       
-      document.getElementById('about-me-form').classList.remove('hidden');
+      //document.getElementById('about-me-form').classList.remove('hidden');
```

Comment:
Probably go ahead and delete comments of legacy code unless they are needed. If they are needed, leave a description as for why.

Reply:
Sorry for the delay! You're right. I deleted it!